src/rebuild.ts

```diff
@@ -109,13 +109,21 @@ export class Rebuilder {
       throw new Error(`Expected a string version for electron version, got a "${typeof this.electronVersion}"`);
     }
 
-    this.ABI = options.forceABI || nodeAbi.getAbi(this.electronVersion, 'electron');
```

Comment:
I have no idea how TypeScript managed to consider this as valid TypeScript, because `forceABI` is a `number` whereas `ABI` is a string.

Reply:
It seems like `nodeAbi` is of type `any` because we're not using `@types/node-abi`. However, I tried adding `@types/node-abi` in and for some reason, it isn't doing anything. Is there some additional tsc configuration needed?

Reply:
This would be why https://github.com/electron/electron-rebuild/blob/dee80faf627d8a3ddc776c73290114ffddba1ef1/src/ambient.d.ts#L2

Delete that line and it should work?